tools/mkclass/classcompiler.cpp

```diff
@@ -151,9 +151,6 @@ static std::string FieldTypeToIcingaName(const Field& field, bool inner)
 	if (field.Attributes & FAEnum)
 		return "Number";
 
-	if (ftype == "bool" || ftype == "int" || ftype == "double")
-		return "Number";
-
 	if (ftype == "int" || ftype == "double")
 		return "Number";
 	else if (ftype == "bool")
```

Comment:
This changes the return value for `ftype == "bool"`.

Reply:
Well, what is the expectation? Do you expect to return `Number` even though it is of type `Boolean`?

Reply:
Well, it should probably return `"Boolean"` for `ftype == "bool"`. The only difference in behavior I found with a quick search through the code is what's returned by the `/v1/types` API.

Nonetheless, this change shouldn't be part of a code style PR but rather its own bugfix PR.

Reply:
By the way, this was introduced by ec7224e3ed2072609c00ba0f27c0a2915d5db852, but I don't see a reason why this change (i.e. returning `"Number"`) should be intended.